spec/features/merchants/items/index_spec.rb

```diff
@@ -29,4 +29,20 @@
 
         expect(current_path).to eq("/merchant/#{@merchant_2.id}/items/#{@item_1.id}")
     end
+
+    it "has a button that can enable/disable items" do
+        visit "/merchant/#{@merchant_2.id}/items"
+
+        within("item-#{@item_1.id}") do
+          expect(page).to have_button("Disable")
+          click_button "Disable"
+          expect(@item_1.status).to eq("Disabled")
+
```

Comment:
If you're having trouble with these attributes not updating, you could try writing `@item_1.reload.status` in your assertion so that rails reloads the model from the database.